src/utils/extractTestResults.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import { DynamoDBRecord } from 'aws-lambda';
+import { DynamoDB } from 'aws-sdk';
+import { DateTime } from 'luxon';
+import { TestResult } from './testResult';
+import { MCRequest } from './MCRequest';
+import logger from '../observability/logger';
+
+/**
+ * This is used to extract the relevant fields from the test record that is
+ * required to be sent to MC in order to  clear prohibitions
+ * @param record
+ */
+export const extractMCTestResults = (record: DynamoDBRecord): MCRequest[] => {
+  try {
+    const data = DynamoDB.Converter.unmarshall(record.dynamodb.NewImage);
+    const mcRequest: MCRequest[] = data.testTypes
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+      .filter((x) => x.testTypeName.toLowerCase().includes('prohibition clearance'))
+      .filter((x) => (x.testResult === ('pass') || x.testResult === ('prs')))
+      .filter(() => (data.vehicleType === 'hgv') || data.vehicleType === 'psv' || data.vehicleType === 'trl')
+      .filter(() => data.testStatus === 'submitted')
```

Comment:
Not 100% necessary, assuming there'll be a terraform PR for these changes, some of the filtering could be done in the dynamo stream rule, just something to keep in mind